app/src/main/java/com/pajato/android/gamechat/main/MainActivity.java

```diff
@@ -164,8 +164,17 @@ public static void processSignIn(@NonNull final Intent intent) {
         View header = navView.getHeaderView(0) != null
             ? navView.getHeaderView(0)
             : navView.inflateHeaderView(R.layout.nav_header_main);
-        View layout = header.findViewById(R.id.currentProfile);
+
+        // Set onClickListeners for the different account icons.
+        View layout = header.findViewById(R.id.alternateAccountIcon1);
+        if (layout != null) layout.setOnClickListener(this);
```

Comment:
Your mother (and Android Studio) have convinced me that these one-liner conditionals were/are ill considered.  They require you to edit code to set a breakpoint is the main complaint and Android Studio support "folding" which provides conciseness.